File: java/springboot3-webflux/src/main/java/com/songday/benchmark/springboot3webflux/model/Book.java
package com.songday.benchmark.springboot3webflux.model;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

import java.time.LocalDateTime;

@Data
@Table("books")
public class Book {
    @Id
    private String id;
    @Column("title")
    private String title;
    @Column("preview")
    private String preview;
    @Column("price")
    private short price;
    @Column("created_at")
    private LocalDateTime createdAt;

}

File: rust/axum-rs/src/model/book.rs
use time::OffsetDateTime;

#[derive(sqlx::FromRow)]
pub(crate) struct BookModel {
    pub(crate) id: String,
    pub(crate) title: String,
    pub(crate) preview: String,
    pub(crate) price: u16,
    pub(crate) created_at: OffsetDateTime,
}
